pkg/cloud/azure/actuators/machine/actuator.go

```diff
@@ -18,25 +18,29 @@ package machine
 
 import (
 	"context"
+	"encoding/base64"
 	"fmt"
 	"strings"
 	"time"
 
+	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2018-10-01/compute"
```

Comment:
We should not import the Azure SDK here. Any methods used should be part of the reconciler or `pkg/cloud/azure/services/*`.